compiler/fir/analysis-tests/testData/resolve/arguments/lambda.kt

```diff
@@ -20,15 +20,15 @@ fun test() {
     bar(x = 1, f = {})
 
     // Bad
-    <!INAPPLICABLE_CANDIDATE!>bar<!> {}
-    <!INAPPLICABLE_CANDIDATE!>bar<!>({})
+    <!NO_VALUE_FOR_PARAMETER{LT}!><!NO_VALUE_FOR_PARAMETER{PSI}!>bar<!> {}<!>
+    <!NO_VALUE_FOR_PARAMETER{LT}!><!NO_VALUE_FOR_PARAMETER{PSI}!>bar<!>({})<!>
 
     // OK
     baz(other = false, f = {})
     baz({}, false)
 
     // Bad
-    <!INAPPLICABLE_CANDIDATE!>baz<!> {}
-    <!INAPPLICABLE_CANDIDATE!>baz<!>() {}
-    <!INAPPLICABLE_CANDIDATE!>baz<!>(other = false) {}
+    <!NO_VALUE_FOR_PARAMETER{LT}!><!NO_VALUE_FOR_PARAMETER{PSI}!>baz<!> {}<!>
```

Comment:
What's causing the difference in marking for LT and PSI?

Reply:
I think it's due to me running the light tree tests but not the PSI tests. Hence, the test data is updated to only have the light tree position updated with the new position strategy. It should be back normal if I run the PSI test as well.

I will update all the test data later after all the needed dependency PRs are merged.